ouroboros-network/src/Ouroboros/Network/Protocol/TxSubmission/Type.hs

```diff
@@ -222,6 +228,8 @@ instance Protocol (TxSubmission txid tx) where
 
   exclusionLemma_NobodyAndServerHaveAgency TokDone tok = case tok of {}
 
+{-# WARNING MsgKThxBye "MsgKTnxBye: should only be used with not yet introduced NodeToNodeV_2" #-}
```

Comment:
Will V2 also include the other outstanding change: include the block size in the Byron headers? We should make sure it does.

Reply:
Yes, we should make both changes at once.